.github/workflows/build.yml

```diff
@@ -11,3 +11,17 @@ jobs:
         run: sudo xcode-select -switch /Applications/Xcode_13.0.app
       - name: Run pod lib lint
         run: pod lib lint
+  spm:
+    name: SPM
```

Comment:
nit: Is the 'name' property ever referenced? maybe worthwhile using the full name rather than the acronym here, just for clarity

Reply:
This name is what shows up on the bottom of the PR (and in the Actions tab in the repo) to visualize the build. I can change it!

Reply:
```suggestion
    name: Swift Package Manager
```